### DISCUSSION TITLE
Username and password?

### DISCUSSION THREAD

**Discussion by gerroon** [vrtmrz/obsidian-livesync#323] [Q&A] [answered]
Hi

I set up using the docker-compose exactly like here https://github.com/vrtmrz/obsidian-livesync/blob/main/docs/setup_own_server.md   I am using Apache to reverse proxy.

The setup does not touch on any user/password What are my user/pass that I need to put in the actual Obsidian plugin settings page?

What do I put here?

![image](https://github.com/vrtmrz/obsidian-livesync/assets/8519469/5689f85d-75ff-4b76-93ad-101a068eb909)

thanks

**Comment by dlphnkck** [comment, marked as answer]
Hello. I apologize for intrusion.
I have no knowledge of Docker, but...


https://github.com/vrtmrz/obsidian-livesync/blob/main/docs/setup_own_server.md
> docker-compose.yml 

```YAML
    environment:
      - COUCHDB_USER=admin
      - COUCHDB_PASSWORD=password
```

So, were you unable to log in?

**Reply by gerroon** [reply]
I do not think it is a Docker thing since CD settings are from the setup.  The custom server does not mention what to do after the  custom server setup, which is where I got stuck

Are you suggesting that the user and the password for Obsidian plugin  is actually the user and pass word of the database admin? 
```
- COUCHDB_USER=admin
- COUCHDB_PASSWORD=password
```
